client/a8c-for-agencies/sections/referrals/primary/referrals-overview/style.scss

```diff
@@ -226,7 +221,9 @@ $data-view-border-color: #f1f1f1;
 	}
 }
 
-// FIXME: This is a temporary fix to make the referral layout columns the same height
+/* FIXME: This is a temporary fix to make the referral
+ * layout columns the same height
+ */
```

Comment:
Lint is complaining about the maximum line length. I have to split it. though not related to the main goal of the PR